### PR TITLE
UCP/CORE: Improve worker creation error flow

### PR DESCRIPTION
**Pull request by brminich**
## What
Improve error flow of worker creation

## Why ?
To avoid segfaults during ifaces cleanup when OOM. Like happened in http://hpc-master.lab.mtl.com:8080/blue/organizations/jenkins/ucx/detail/ucx/2828/pipeline/109
```
[2019-11-24T19:01:40.369Z] [ RUN      ] shm_ib/test_ucp_stream.send_recv_data_recv_iov/0 <shm,ib>

[2019-11-24T19:01:40.369Z] [boo31:8327 :0:8327] Caught signal 11 (Segmentation fault: address not mapped to object at address (nil))

[2019-11-24T19:01:40.933Z] 

[2019-11-24T19:01:40.933Z] /scrap/jenkins/workspace/ucx-2/contrib/../src/ucp/core/ucp_worker.c: [ ucp_worker_close_ifaces() ]

[2019-11-24T19:01:40.933Z]       ...

[2019-11-24T19:01:40.933Z]      1036     UCS_ASYNC_BLOCK(&worker->async);

[2019-11-24T19:01:40.933Z]      1037     for (iface_id = 0; iface_id < worker->num_ifaces; ++iface_id) {

[2019-11-24T19:01:40.933Z]      1038         wiface = worker->ifaces[iface_id];

[2019-11-24T19:01:40.933Z] ==>  1039         if (wiface->iface != NULL) {

[2019-11-24T19:01:40.933Z]      1040             ucp_worker_iface_cleanup(wiface);

[2019-11-24T19:01:40.933Z]      1041         }

[2019-11-24T19:01:40.933Z]      1042     }
```

## How ?
print exact error when failed to allocate memory for ifaces and set num_ifaces to 0 (to avoid looping thru them during cleanup)

### CODE REVIEW

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 02-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/4d8d2f4cadf844ba2845e08b9b933f5e4d872baa/_main) | :x: FAILURE
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/2d29564192641e1d7e2a8c7996fb941123f5cbe0/r-vmb-ppc-jenkins_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/ae0d1f02c3f727fe5a10d9a7e00f19b978521c77/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/a94139cd9c5ec858ca7f8f50a30274681e992767/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/e73c9b604e9efb0b6e687ae4c7af7c3494858f05/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/d6655e88fc2393844914719c86db3821a353cc11/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/c093f5737768208b2a8232427b9005425701b462/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/99d6af03a51ef60db1b5c86f39007e9c30fe572b/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/d0e98e1fa239e80cb801d19143a8118006595874/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/d8c0c01a8d13f5509683d8f4de34a3528480d7dd/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/d338c8edb295b8d9e3792da690cf2b56b732e896/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/c13e31d69f29dd8aa06788c3d053e41e981e7ea9/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/42420d485eca59dacc71c10fb9bce7ae554c98a2/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/a0081b7e32011f04869d98851df78569a1a98731/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/722324e72d390c83256ed37b742806ab31f2f83d/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/47f22cae00d10c23b07a2c8c42e3e76d825d627b/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/fd8079283bb24512d4ef30a4f430dc1bf4ebc422/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/53b55e87602d16ce493c2950d5eccec9f5e6b73d/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/418dc9f92d0155f0e5e68f50203ccd2bda3422c5/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/9c0733283a7d961b463cbb95d6fc4331326f2fc4/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/b06a3f64bd046b64afd2a5660872ff12be3516cb/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/74dd20911d21eedf1da61a86b076fdda4acc874a/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/fcdf2d85a7e4eb422e2e0935f4ca3ebd5f04d2e8/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/31873048fb70f8a84e8e4553c97c602edff0cbbc/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/f39cd443c17edcec376d6849c08af539/raw/9f6f67c184438ac87b48adb59857142b170e2051/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
issue is #4331 
 
bot:mlx:retest

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 02-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/a10c1492e660bd4d0f951fdfd92c9e920ea8eec8/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/53bc24136c993980ec5ff2b9bb8e33ef141f51f9/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/584db02fba86e0deef4a63a76b330feeef3283bd/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/fb8c5eac221f81ac837a0634b20fc2331725ae79/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/b71f8f812539da1bc47ba8a6af5dc4872ac52c97/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/e8154473da7af8e27e47333f10fe513f5054a8f7/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/ee5a355c100b85e0312be24ffb49a3d240f86d85/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/1f2f62347a00a8aac6867c5e2e24a80e23bb8bb6/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/c8b5c6011b271c4d6e3b1f63763f17ddf01978df/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/45e94014476b37ab8766d8c929edd497063d4aae/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/a2a098e39b66f1b8a2ac70ec0c43a50a671da31a/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/25e8faeabf8010103f48795c963fdbe970508bb8/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/d83570a0fe75ec3a72b6ec55b7d6a36ce1430b45/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/5dc8a8c24d66b50eb430e517a61183448f880402/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/9491674ef439ce3ae4a60a133c61507587176f93/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/fdbceaccdc7fd2b28a5a47e048b891dc47c4269a/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/7244e74b29c3b0b4f0a611d090c21dff02094458/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/a8cae2bcf23ffdd134d9e5c38ce9171af7b86369/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/83880ce2080f4f78634f9f18388d3fae21113f12/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/3b23139af5d5f8a8aad8ed3da0e32b1696063ba9/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/664cf38bfda332ad27fb618ac929970f87a4685a/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/31b7e9873fbb194327f9b81d3a5c65234450a968/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/08abbad12fc4b7a5c3de4f290e6b8bb7e966e671/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/88d809a16b70732b8d8e719c63c8c5828d8a4ea7/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/84f54a99bba478dc6e89af3af83af66c/raw/b91ca24bc0eafa3689a68534dcd8f96ec9de969b/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
@hoopoepg, plz review

File: src/ucp/core/ucp_worker.c [hunk 1]
```diff
@@ -963,6 +965,8 @@ static ucs_status_t ucp_worker_add_resource_ifaces(ucp_worker_h worker)
     worker->ifaces = ucs_calloc(worker->num_ifaces, sizeof(*worker->ifaces),
                                 "ucp ifaces array");
     if (worker->ifaces == NULL) {
+        ucs_error("failed to allocate worker ifaces");
+        worker->num_ifaces = 0;
```
[the remark this answers is not in the document]
Comment (brminich) [reply src/ucp/core/ucp_worker.c]: i need proper `num_ifaces` to allocate an array of them on line 965. Can use tmp var and assign ` worker->num_ifaces` to it after successful allocation.

[the remark this answers is not in the document]
Comment (yosefe) [reply src/ucp/core/ucp_worker.c]: > Can use tmp var and assign ` worker->num_ifaces` to it after successful allocation.

Yep

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 5 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 03-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/848a1f5c08d524ddabf9f19532f07f38c2657cd6/_main) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/f8510751947aead34092c342b8ab3fdf7c99b241/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/f4dde0d782afd841842a49447fae3856ae55dc2f/hpc-test-node-legacy_W1) | :x: FAILURE
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/e851977ea04090a07491eb779e89c47128954ce2/hpc-test-node-legacy_W2) | :x: FAILURE
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/b7878743f0079e52d8015d9985a3bba895c262fa/hpc-test-node-legacy_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/12ee7586e8bea9fb7728812a67344615e24d412c/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/0e84d234f4fe2323e31abdca0aad3e474defd341/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/1d30bd97b929175f585059e02299961cd414687c/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/51c45bc5fdea9e013cb534d6d963bf20df22fea2/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/62b7bf80f95c25aedef1f8964ffe51aad58c0659/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/fd6d6b2388775daf9d88d77a59ee90c2b1f7e4dc/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/eb40e8423964a06e4ce1a7c012dbef2f0d45ee3d/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/6a8470404999f27b98e8211e06092694edfe708e/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/140495a6d9686e4cb4e34db04faf69cc31cf073c/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/96ed8bab6b1d8ad0827967ae820fff2b05abaa7c/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/efd67fa4a22f953c9a5a6edccb3b203d7c9eb5e8/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/7d8a1a11c9b08074c35141faede50b270b982ed3/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/3853e47b8d20d96978e353f4b593e91e638237c7/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/f971e99f41fea2ee35d32036f62ca5d0ef1270b5/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/b6ef655258f68d960174a9f57767682174b82f51/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/5118a203d0c87da32d782041623a66cfc5f55572/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/eb19587a4d81f1eaa2e65d5aac47b5c66e581fdf/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/781851d480d94f0696a1137ddf8126b2db262d23/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/992d60e8c50485a52b6db398942a9a7fe8ebab29/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9fb4e4eddf68c14d947d8fa17413873b/raw/1660cb3582ea9da4d108a0716c9b9bf9577ef348/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
#4512 
bot:mlx:retest

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 03-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/e2cd9b7505788cee788277fedccfd97b27190860/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/e9a60e3b9f84e2bcd73c087279de2dc7a5558f34/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/e88ab06f63e24b6dbe52556d14989e35c0acc526/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/5e781b7642e05206947580dd5ff2444a05e1fad5/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/6735cd38f9c6822ada2aad1957cb5a9f02b0d40c/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/f24cfc24d3429edc4802ae362bbf9549c8509b9e/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/126b4cf5fe93e73d1a6782965e9f7909d4a6482a/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/add09a250275514488fc872668c13764e5b75363/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/011d606316f21157b145f78774fe449d9ba45ea3/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/be882b519e293f8fae5d45badae2b8036ab19f88/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/8b8a377330582d545a129cdfa37a55c8d323340c/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/8f327af712bf41f28224c5527b7d8fa605a7f9b2/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/79257ef22384b9a2147f61c2dbbab6b49ada3c15/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/c6fc1c20e386d0e47fabd29726ddb84c4ee1cf4a/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/bc6c769272cab4bd3bc0ff89f0c20286b6d4cc7b/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/edc5ede0c8502920283637dec101f2272f2beeda/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/bda2849fbb2b682bbc42fdc97edf75ae87120d2c/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/72d280b7b33b6b84cb4fb9c056d2f545c9a29fab/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/9195be319617ed02280b5b2ac238ead9e664ed58/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/9f6c990b48f314a3725528fc2bd127183fc24247/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/c2ec507aa1972f2c71ded4a25349c504dae4c062/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/1607a5bb6383a2a0bf2da92e9fd5f573ae995d82/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/ec710baa687392ac9d89347e9584733afe980ab9/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/0e46d34d00f7f80e8635bd2c41f9d447003f31ec/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/22b171b2c0f3bc625e3f9f47eea7bcb0/raw/98cec2ca8edb23d2b8e2d0932d6f3c84f21f528c/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
OOM
bot:pipe:retest

**Comment by brminich** [comment]
@yosefe, ok to merge?

**Pull request merged**